blocks/init.php

```diff
@@ -0,0 +1,238 @@
+<?php
+/**
+ * Blocks Initializer
+ *
+ * Enqueue CSS/JS of all the blocks.
+ *
+ * @since   1.0.0
+ * @package CGB
+ */
+
+// Exit if accessed directly.
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+
+/**
+ * Register the block on the init hook.
+ *
+ * Note: We need to pass the attributes in order for the ServerSideRender to work
+ */
+function register_author_avatars_block() {
+	register_block_type(
+		__DIR__  . '/build/show-avatar', // Path to the block.json file to load the files we need.
+		array(
+			'render_callback' => 'callback',
+			'attributes'      => array(
+				'user_id'          => array(
+					'type' => 'number',
+				),
+				'size'             => array(
+					'type' => 'number',
+				),
+				'email'            => array(
+					'type' => 'string',
+				),
+				'alignment'        => array(
+					'type' => 'string',
+				),
+				'link'             => array(
+					'type' => 'string',
+				),
+				'display'          => array(
+					'type' => 'object',
+				),
+				'sort_avatars_by'  => array(
+					'type' => 'string',
+				),
+				'sort_order'       => array(
+					'type' => 'string',
+				),
+				'limit'            => array(
+					'type' => 'number',
+				),
+				'bio_length'       => array(
+					'type' => 'number',
+				),
+				'page_size'        => array(
+					'type' => 'number',
+				),
+				'min_post_count'   => array(
+					'type' => 'number',
+				),
+				'hidden_users'     => array(
+					'type' => 'string',
+				),
+				'whitelist_users'  => array(
+					'type' => 'string',
+				),
+				'background_color' => array(
+					'type' => 'string',
+				),
+				'font_color'       => array(
+					'type' => 'string',
+				),
+				'block_style'      => array(
+					'type' => 'string',
+				),
+				'className'        => array(
+					'type' => 'string',
+				),
+				'border_radius'    => array(
+					'type' => 'number',
+				),
+				'sort'             => array(
+					'type' => 'string',
+				),
+				'sort_by'          => array(
+					'type' => 'string',
+				),
+				'role'             => array(
+					'type' => 'object',
+				),
+				'blogs'            => array(
+					'type' => 'object',
+				),
+				'border_size'            => array(
+					'type' => 'number',
+				),
+				'border_color'            => array(
+					'type' => 'string',
+				),
+			)
+		)
+	);
+
+}
+add_action( 'init', 'register_author_avatars_block' );
+
+function callback( $attributes, $content ) {
```

Comment:
Next steps is to remove this function and use a `render.php` instead in the block.